nwplus/nwhacks2019
# Functions/import mentors

:tickets: **Ticket(s)**: Closes #

---

## :construction_worker: Changes

A brief summary of what changes were introduced.

## :thought_balloon: Notes

Any additional things to take into consideration.

## :flashlight: Testing Instructions

1. Export the mentor rsvp form as csv from google form
2. Go to the "Storage" tab on "nwhacks-2019-dev" firebase console
3. Upload the csv and copy Download url from the right panel
4. Delete the collection `mentors`
5. Go to `admin/tasks` and paste the download url into text field
6. Click on the import mentors button
7. Check that the mentors have been imported with the corresponding fields
